### PR TITLE
Add some unittest for celery queue scoring system

### PR DESCRIPTION
**Pull request by Ladsgroup**
Bug: T212090

### CODE REVIEW

File: ores/scoring_systems/tests/test_celery_queue.py [hunk 1]
```diff
@@ -29,3 +34,59 @@ def test_celery_queue():
...
+
+    assert len(list(scoring_system.task_tracker.tasks.keys())) == 1
+    assert list(scoring_system.task_tracker.tasks.values()) == ['Task ID']
+    key = list(scoring_system.task_tracker.tasks.keys())[0]
+
+    # We should not assert the exact key as it's internal but we should assert it has the important bits
+    assert 'fakewiki' in key
+    assert 'revid' in key
+    assert '0.0.1' in key
+    assert '123' in key
```
Comment (halfak) [inline ores/scoring_systems/tests/test_celery_queue.py]: Good idea.  I like it.

**Comment by halfak** [comment]
Sorry to add some comments late.  Apparently I'd started a review and forgot to submit them.  All of my comments are present now.

File: ores/scoring_systems/tests/test_celery_queue.py [hunk 2]
```diff
@@ -29,3 +34,59 @@ def test_celery_queue():
...
+        'injection_caches': {},
+        'ip': None,
+        'model_info': None
+    }
+
+    scoring_system._lock_process(['revid'], 123, ScoreRequest.from_json(request), None, 'Task ID')
+
+    assert len(list(scoring_system.task_tracker.tasks.keys())) == 1
+    assert list(scoring_system.task_tracker.tasks.values()) == ['Task ID']
+    key = list(scoring_system.task_tracker.tasks.keys())[0]
```
[the remark this answers is not in the document]
Comment (Ladsgroup) [reply ores/scoring_systems/tests/test_celery_queue.py]: I really wanted to go with something better but the celery queue itself is not very testable in that regard. I need to fix that in other iterations IMO.

File: ores/scoring_systems/tests/test_celery_queue.py [hunk 3]
```diff
@@ -1,5 +1,10 @@
 import celery
+from revscoring.extractors import OfflineExtractor
 
+from ores.score_request import ScoreRequest
+from ores.scoring.models import RevIdScorer
+from ores.scoring_context import ScoringContext
+from ores.task_tracker import InMemoryTaskTracker
```
[the remark this answers is not in the document]
Comment (Ladsgroup) [reply ores/scoring_systems/tests/test_celery_queue.py]: Fixed. Thanks.

**Pull request merged**